src/components/Tests/Calculator.test.js

```diff
@@ -0,0 +1,7 @@
+import Renderer from 'react-test-renderer';
+import Calculator from '../Calculator';
+
+it('Renders Calculator component', () => {
+  const tree = Renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] You have installed the react testing library, but still need to add tests using it. Kindly write tests simulating user interactions like firing events like clicks and performing calculations on the component. You can link to this tutorial on how to set up tests using the testing library: https://www.robinwieruch.de/react-testing-library/

Reply:
- [ ] Test added to simulate an event interaction and perform a calculation on component